### DISCUSSION TITLE
Feature Request - Add OpenRouter.ai API Option

### DISCUSSION THREAD

**Discussion by damian-gith** [cline/cline#282] [Feature Requests]
The title say everything, can we add OpenRouter.ai API option and then choose our models to use.

**Comment by entrepeneur4lyf** [comment]
I'm pretty sure it's had this feature for quite some time. I've been using Openrouter in ClaudeDev as a way to defeat daily limits with Claude. He even added support for prompt caching for OpenRouter recently.

**Comment by elemental-mind** [comment]
This discussion should be closed
